app/models/manageiq/providers/storage_manager/storage_manager_mixin.rb

```diff
@@ -1,4 +1,4 @@
-module StorageManagerMixin
+module ManageIQ::Providers::StorageManager::StorageManagerMixin
```

Comment:
@jerryk55 @roliveri Will rename it to CinderManagerMixin and put it to app/models/mixins.